analysis_plan.md

```diff
@@ -0,0 +1,20 @@
+# First analysis plan
+
+This is the first analysis plan.
+
+It is provisional.
+
+## Reading to do
+
+* Read about Nipy diagnostics
+* Consider Principal Component Analysis (good tutorial?)
+* Consider Independent Component Analysis (e.g. FSL)
+* Your ideas here.
+
+## Analysis plan
+
+* Build simplest possible draft using e.g. standard deviation
```

Comment:
Good plan. Let's not get too complicated all at once.

Reply:
Thanks!